packages/recozik-web/src/recozik_web/auth_service.py

```
@@ -0,0 +1,164 @@
+
+
+def hash_password(password: str) -> str:
+    """Return Argon2id hash for the given password."""
+    return ph.hash(password)
+
+
+def verify_password(password: str, hashed: str) -> bool:
+    """Return True if password matches the stored hash."""
+    try:
+        return ph.verify(hashed, password)
+    except Exception:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)